### ISSUE TITLE
loss not converged

### ISSUE THREAD

**Issue by kimwin2** [Lucky-Lance/SPP#1]
Firstly, Thanks for your great effort about this project. 

I have re-implemented this project. I used the WANDA code to perform unstructured pruning by 50% and obtained a checkpoint. After that, I read your project's README and conducted re-training using the same method and configuration. However, the loss was larger than 100000 from the beginning. What could be the problem? 

And is there any way to use your code to start with the fine-tuning of WANDA?
